db_stress_tool/db_stress_test_base.cc

```diff
@@ -114,9 +114,9 @@ std::shared_ptr<Cache> StressTest::NewCache(size_t capacity,
   }
 
   if (FLAGS_cache_type == "clock_cache") {
-    auto cache = NewClockCache(static_cast<size_t>(capacity), FLAGS_block_size,
-                               num_shard_bits, false /*strict_capacity_limit*/,
-                               kDefaultCacheMetadataChargePolicy);
+    auto cache = ExperimentalNewClockCache(
```

Comment:
I missed this in the previous PR---in our internal tests we want to use the new ClockCache.